tools/build-kani/src/sysroot.rs

```diff
@@ -106,7 +106,12 @@ pub fn build_lib() {
 
     // Collect the build artifacts.
     let artifacts = build_artifacts(&mut cmd);
-    let _ = cmd.wait().expect("Couldn't get cargo's exit status");
+    let exit_status = cmd.wait().expect("Couldn't get cargo's exit status");
+    // exit_ok is an experimental API where we could do .exit_ok().expect("Build failed")
+    // Until it is stable:
```

Comment:
Please backticks when writing code or function names in comments, e.g.:
```suggestion
    // `exit_ok` is an experimental API where we could do `.exit_ok().expect("Build failed")`
    // Until it is stable:
```
Regardless, you could just say we could do the same with `exit_ok` (i.e., without the example) but we avoid it since it's experimental.

Reply:
Done.